packages/snaps-controllers/src/services/iframe/test/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html>
+<head>
+  <title>Sandbox test</title>
+</head>
+<body>
+<!--
+This file is used to test the sandboxing of the iframe. It is used by the
+`IframeExecutionService.test.browser.ts` test.
+-->
+<script>
+  try {
+    top[0].document;
```

Comment:
`top[0].document; // should resolve to snaps iframe`

Reply:
actually i'd love it also if we create the iframe with an `id` and then look it up based on its `id` rather than accessing it using `window[n]` to be more specific:

`top.getElementById(<ID>).contentWindow.document; // should resolve to snaps iframe` for example is better